CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.1.0 - [unreleased]
+
+### `Added`
+
+### `Fixed`
```

Comment:
```suggestion
```suggestion
- [#416](https://github.com/nf-core/funcscan/pull/416/) - Post-release version bump

```### `Fixed`
```

Reply:
but if you don't include this usually it's a :+1: from me then